CWOD-Exalted/sheet.json

```
@@ -0,0 +1,8 @@
+{
+  "html": "EXvWOD.html",
+  "css": "EXvWOD.css",
+  "authors": "Chris F.",
+  "roll20userid": "444248",
+  "preview": "EXvWOD.png",
+  "instructions": "# Exalted vs World of Darkness character sheet \r \r **For use in the classic World of Darkness** \r\r # Exalted vs World of Darkness character sheet\r\rBased on the (Exalted vs World of Darkness)[https://www.patreon.com/posts/exalted-vs-world-23661124] fan supplement by Holden Shearer.\r\r\r\r## Features\r\r* *All attributes, skills, and so on are fill from the left.* \r\r* Health boxes toggle through bashing (/), lethal (X), and aggravated (*).\r\r* Essence and Willpower boxes toggle between unspent ( ) and spent (X).\r\r* Excess Essence boxes are blacked out based on Essence (dots) and Exalted Type.\r\r* Ox-Body Technique is its own attribute, and determines how many boxes to render.\r\r* Roll buttons for soak, initiative, dodge, parry, block, attacks and damage.\r\r* If there's something you want added, or you'd like to report a bug, please contact the author of this sheet."
```

```suggestion
  "instructions": "# Exalted vs World of Darkness character sheet \r \r **For use in the classic World of Darkness** \r\r # Exalted vs World of Darkness character sheet\r\rBased on the [Exalted vs World of Darkness](https://www.patreon.com/posts/exalted-vs-world-23661124) fan supplement by Holden Shearer.\r\r\r\r## Features\r\r* *All attributes, skills, and so on are fill from the left.* \r\r* Health boxes toggle through bashing (/), lethal (X), and aggravated (*).\r\r* Essence and Willpower boxes toggle between unspent ( ) and spent (X).\r\r* Excess Essence boxes are blacked out based on Essence (dots) and Exalted Type.\r\r* Ox-Body Technique is its own attribute, and determines how many boxes to render.\r\r* Roll buttons for soak, initiative, dodge, parry, block, attacks and damage.\r\r* If there's something you want added, or you'd like to report a bug, please contact the author of this sheet."
```
Think you did an unintended switcheroo on the link, this way it should work as intended.